uaa/src/main/resources/uaa-docker.yml

```diff
@@ -608,4 +608,68 @@ uaa:
 #  host: localhost
 #  port: 2525
 #  user:
-#  password:
\ No newline at end of file
+#  password:
+
+servlet:
+  session-store: database
+
+encryption:
+  active_key_label: CHANGE-THIS-KEY
+  encryption_keys:
+  - label: CHANGE-THIS-KEY
+    passphrase: CHANGEME
+
+issuer:
+  uri: http://localhost:8080/uaa
+
+#The secret that an external login server will use to authenticate to the uaa using the id `login`
+LOGIN_SECRET: loginsecret
+
+jwt:
+  token:
+    signing-key: tokenKey
+
+login:
```

Comment:
We actually have login already defined above - with serviceProviderKey commented? Maybe instead of using properties from the required_configuration.yml, we should look at what is missing uaa-docker.yml when compared with uaa.yml?

Reply:
I could definitely make uaa-docker.yml prettier; keeping it as is and adding missing configuration to the end was done so diff is easier.

Reply:
uaa-docker.yml is based on the upstream [uaa.yml](https://github.com/GESoftware-CF/uaa/blob/master/uaa/src/main/resources/uaa.yml), which is provided as a configuration example, plus [required configuration](https://github.com/GESoftware-CF/uaa/blob/master/scripts/cargo/uaa.yml), plus our [customizations](https://github.com/GESoftware-CF/uaa/blob/predix_extensions_75.6.0/uaa/src/main/resources/uaa-docker.yml#L555-L563)

Reply:
I meant there are two login sections/keys now in the uaa-docker.yml - not sure how parsing it will work. Yeah, think its good to clean it up so that its clear what we are adding on to uaa.yml.

Reply:
Updated commit to combine the yaml file keys without any duplicate keys and to base it off the current uaa.yml file from upstream.